cypress_test/integration_tests/multiuser/writer/annotation_spec.js

```diff
@@ -3,6 +3,57 @@
 var helper = require('../../common/helper');
 var desktopHelper = require('../../common/desktop_helper');
 
+describe(['tagmultiuser'], 'Annotation Visibility Tests', function () { // WIP WIP WIP WIP WIP
+	beforeEach(function() {
+		helper.setupAndLoadDocument('writer/annotation.odt',true);
+		cy.viewport(2800, 600);
+		desktopHelper.switchUIToNotebookbar();
+		cy.cSetActiveFrame('#iframe1');
+		cy.cGet('#sidebar-dock-wrapper').should('be.visible');
+		desktopHelper.sidebarToggle();
+		desktopHelper.selectZoomLevel('120', false);
+		cy.cSetActiveFrame('#iframe2');
+		cy.cGet('#sidebar-dock-wrapper').should('be.visible');
+		desktopHelper.sidebarToggle();
+		desktopHelper.selectZoomLevel('120', false);
+	});
+
+	/*
+		when the document is opened, two 'resize' events are fired,
+		but both of them fail because by then comments haven't been
+		loaded. then the sidebar shows up and invalidates the view,
+		which fires the 'resize' event and by then we have the comments
+		and we bring them into the view.
+
+		but when the sidebar is hidden and the document is reloaded,
+		the third 'resize' never happens and thus the document remains
+		centered. this is a problem as if the window is not wide enough,
+		the comments will be only half visible.
+	*/
+	it('Center Document On Reload without Sidebar Visible', function() {
+		cy.cSetActiveFrame('#iframe1');
+		desktopHelper.insertComment();
+
+		cy.cSetActiveFrame('#iframe2');
+		desktopHelper.insertComment();
+
+		cy.cSetActiveFrame('#iframe1');
+		/*
+			although this reloads the first client, this
+			still triggers the sidebar which fires the
+			resize event and "REDUCES THE DOCUMENT ZOOM FOR SOME REASON (WHY? BUG!)"
+
+			HOW CAN I RELOAD AND NOT TRIGGER THE SIDEBAR?
```

Comment:
reloadDocument helper: https://github.com/CollaboraOnline/online/blob/master/cypress_test/integration_tests/common/helper.js#L128

did you try to:
- close sidebar, then reload? is the state remembered?
here is code about sidebar initialization on load:
https://github.com/CollaboraOnline/online/blob/31c4756143e7d9f62be2a6461743ebfa4c81c059/browser/src/control/Control.UIManager.ts#L628
- maybe we can temporarily write to prefs
- so it will not show by : `var showSidebar = this.getBooleanDocTypePref('ShowSidebar', true);`

if not, then we could use UI customization parameters possibly: `UIDefaults` param to the initial POST